Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-09-13
Initial Package Version: 2.7
Origin: Gentoo
Description: Missing Script

diff -Naur cracklib,2.7.orig/util/create-cracklib-dict cracklib,2.7/util/create-cracklib-dict
--- cracklib,2.7.orig/util/create-cracklib-dict	1970-01-01 00:00:00.000000000 +0000
+++ cracklib,2.7/util/create-cracklib-dict	2003-09-13 09:57:03.000000000 +0000
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [ -z "$*" ]; then
+    echo "Usage:"
+    echo "  /usr/sbin/create-cracklib-dict wordlist ..."
+    echo
+    echo "This script takes one or more word list files as arguments"
+    echo "and converts them into cracklib dictionaries for use"
+    echo "by password checking programs. The results are placed in"
+    echo "/usr/lib/cracklib_dict.*"
+    echo
+    echo "Example:"
+    echo "/usr/sbin/create-cracklib-dict /usr/dict/words"
+else
+    /usr/sbin/mkdict $* | /usr/sbin/packer /usr/share/cracklib/pw_dict
+fi
